index.js

```diff
@@ -129,7 +180,19 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  let movieFilteredByGenre = [];
```

Comment:
Ooopsie! Looks like we got some copy 🍝 pasta mishaps here. I think we meant to name this variable something else?